Bookish/.gitignore

```diff
@@ -5,6 +5,9 @@ project.lock.json
 *.pyc
 nupkg/
 
+#API Keys
+*/Keys.cs

```

Comment:
That's one way of doing it, well done not committing secrets!

ASP.NET normally uses the appSettings.json file for configuration, if you have secret keys you normally make an appSettings.Development.json, .gitignore that and then put the keys in there.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0